src/http/states/readRequestLine/ParseVersion.cpp

```diff
@@ -0,0 +1,79 @@
+#include "ParseVersion.hpp"
+
+#include <client/Client.hpp>
+#include <http/Request.hpp>
+#include <http/StatusCode.hpp>
+#include <http/abnfRules/generalRules.hpp>
+#include <http/states/readRequestLine/ReadRequestLine.hpp>
+#include <libftpp/string.hpp>
+#include <utils/Buffer.hpp>
+#include <utils/BufferReader.hpp>
+#include <utils/abnfRules/LiteralRule.hpp>
+#include <utils/abnfRules/RangeRule.hpp>
+#include <utils/state/IState.hpp>
+
+#include <cctype>
```

Comment:
```suggestion
#include <ctype.h>
```
`::isdigit` is used, not `std::isdigit`.